profile/srap-profile.md

```diff
@@ -96,6 +94,17 @@ This table contains all the proposed new or changed element, and DC Terms elemen
 | Title             | http://purl.org/dc/terms/title                | 1     | Free text, titles in different languages provided separately |
 | Type              | http://purl.org/dc/terms/type                 | 0...1 | A term from the DCMI Type vocabulary or from the the [COAR resource type vocabulary](http://vocabularies.coar-repositories.org/pubby/resource_type) |
 
+## Roles 
+
+Creators and contributors may have different roles. SRAP uses the Library of Congress relators to express roles a person or organization may have (editor, funder, etc.). If there is a need to describe an additional role or roles are needed, a request to register a new code will be sent to LC.    
```

Comment:
Agreed in 2022-10-11 meeting on the wording:

SRAP recommends the use of a controlled list of roles such as the Library of Congress relators ...

Reply:
Done